### PR TITLE
fix: post-ship website fixes and enhancements

### PR DESCRIPTION
**Pull request by orvn**
Various fixes and enhancements for the new website, which was shipped to production last week.
Most are edge-case or visual bug related.

- [x] Overflow of non-breaking file names
- [x] Docs footer contact area being condensed to one column on mobile
- [x] When a file is recently uploaded, it SP field appears as `null`
- [x]  Re-add Countly tracking on 404 page
- [x]  Add the error dialog to token deletion

### CODE REVIEW

**Comment by orvn** [comment]
Closes #1131, #1127, #1146

File: packages/website/pages/404.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 
-const Custom404 = () => (
-  <main className="page-404">
-    <BlockBuilder id="error_section-1" subsections={blocks} />
-  </main>
-);
+export default function Custom404() {
+  useEffect(() => {
+    countly.trackEvent(countly.events.NOT_FOUND, {
+      path: '/404',
```
Comment (gobengo) [inline packages/website/pages/404.js:12]: Might it be useful to get this value from `useRouter()`? https://nextjs.org/docs/api-reference/next/router#userouter

**Review by olizilla — APPROVED** [review]
Minor tweaks suggested

- send 404 path to countly as suggested by @gobengo 
- revert change to package-lock.json
- review issue #1127 - the word break is fixed but the vertical space in the feature boxes remains. It may be intentional that the text is bottom aligned in their but please comment either way.
Screenshot 2022-03-31 at 14 48 34


Otherwise, no need to block on further reviews for this PR... merge when ready.

**Comment by orvn** [comment]
> Minor tweaks suggested
> 
>     * send 404 path to countly as suggested by @gobengo
> 
>     * revert change to package-lock.json
> 
>     * review issue #1127 - the word break is fixed but the vertical space in the feature boxes remains. It may be intentional that the text is bottom aligned in their but please comment either way.
> 
> Otherwise, no need to block on further reviews for this PR... merge when ready.

Sounds good!

- Making those changes
- The asymmetric vertical alignment was indeed intentional, I'm good to change it if there are strong feelings that it appears out of place though. Will comment on the original issue #1127.

**Pull request merged**